scripts/buildkite/golint.sh

```diff
@@ -4,16 +4,17 @@ set -ex
 
 make tidy
 make go-generate
+make copyright
 make fmt
```

Comment:
fmt will fail if copyright is not up to date, so there's not really any point in fmt-then-copyright.